saathi.sql

```diff
@@ -1,31 +1,3 @@
--- phpMyAdmin SQL Dump
--- version 4.5.0-dev
--- http://www.phpmyadmin.net
---
--- Host: localhost
--- Generation Time: Mar 26, 2015 at 09:13 PM
--- Server version: 5.5.41-0ubuntu0.14.04.1
--- PHP Version: 5.5.9-1ubuntu4.7
-
-SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
-SET time_zone = "+00:00";
-
-
-/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
-/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
-/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
-/*!40101 SET NAMES utf8 */;
-
---
--- Database: `saathi`
---
-
--- --------------------------------------------------------
-
---
--- Table structure for table `assets`
---
-
```

Comment:
@ParthKolekar Why are you removing the comments. What harm is it doing?? Please revert these changes.